tests/unit/storage/test_flattenedstorage.py

```diff
@@ -945,8 +963,21 @@ def test_to_pandas(self):
                         f"Element {i} in dataframe not equal to original: {elem_df}!={elem_st}!",
                     )
 
-        dfe = store.to_pandas(explode=True)
+        try:
+            dfe = store.to_pandas(explode=True)
+        except:
+            # Regression test
+            self.fail(
+                "to_pandas must not fail when multidimensional arrays are present!"
+            )
```

Comment:
I am a bit confused about the `try except` part. I agree that it gives a nicer error message but at the same time it is not as readable.